src/dotnet-install.ps1

```diff
@@ -32,6 +32,8 @@
     - latest - most latest build on specific channel
     - 3-part version in a format A.B.C - represents specific version of build
           examples: 2.0.0-preview2-006120, 1.1.0
+.PARAMETER Internal
+    Set this parameter to access internal builds. It might be nessesary also to use FeedCredential parameter with it.
```

Comment:
My suggestion:
Set this switch to download internal builds. It is necessary to use FeedCredential option with this.